test/ibmq/test_ibmq_job.py

```diff
@@ -400,33 +415,42 @@ def test_get_jobs_filter_counts_backend_service(self, provider):
             self.assertTrue(any(cresult.data.counts.to_dict()['0x0'] < 500
                                 for cresult in result.results))
 
-    @requires_device
-    def test_get_jobs_filter_date_backend(self, backend):
+    @requires_provider
+    def test_get_jobs_filter_date_backend(self, provider):
```

Comment:
I don't think it's all that necessary to have a separate test case for `backend.jobs()`, since it just calls `backendservice.jobs()` (but I suppose there is no harm since it's quick - your decision). It'd be more useful to have a test case that specifies both datetime as keyword and inside `db_filter`, for example `start_date=past_month, db_filter={'creationDate': {'lt': past_two_month}}`